setup.py

```diff
@@ -113,11 +118,13 @@ def run(self):
         + dew_requires
         + gridlabd_requires
         + ephasor_requires
-        + cyme_requires,
+        + cyme_requires
+        + synergi_requires,
         "extras": extras_requires,
         "cyme": cyme_requires,
         "dew": dew_requires,
         "ephasor": ephasor_requires,
+        "synergy": synergi_requires,
```

Comment:
Should this be spelt `synergy` or `synergi`?

Reply:
```synergi``` (at least it's the way I think it should be spelt but I keep doing the mistake....). Thanks for spoting this, I'll change it asap!